frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx

```diff
@@ -176,14 +214,29 @@ export const FieldRowContainer = ({
                   colorScheme="secondary"
                   spacing={0}
                 >
+                  <IconButton
+                    aria-label="Close field settings"
+                    icon={<BiCog fontSize="1.25rem" />}
+                    onClick={setToInactive}
```

Comment:
ohya this should be disabled state when queries are loading too

Reply:
09c6c655